container-tests/tests/build.gradle

```diff
@@ -0,0 +1,33 @@
+
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+jar { enabled = false }
+
+dependencies {
+  testImplementation 'junit:junit'
+  testImplementation 'org.apache.logging.log4j:log4j-slf4j-impl'
+  testImplementation 'org.assertj:assertj-core'
+  testImplementation 'org.awaitility:awaitility'
+  testImplementation 'org.testcontainers:testcontainers'
+  testImplementation 'org.web3j:core'
+  testImplementation 'org.web3j:quorum'
+}
+
+test.enabled = false
+
+task containerTests(type: Test) {
+  description = 'Runs GoQuorum <> Besu container tests.'
```

Comment:
this is where you need to specify dependsOn root project's distDocker ... and re-calculate docker image. Here, you can either set system property or environment variable which will be visible to all tests.